packages/client3/.storybook/DocsContainer.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { DocsContainer as BaseContainer } from "@storybook/addon-docs/blocks";
+import { useDarkMode } from "storybook-dark-mode";
+import { themes } from "@storybook/theming";
+
+export const DocsContainer = ({ children, context }) => {
+  const dark = useDarkMode();
+
+  return (
+    <BaseContainer
+      context={{
+        ...context,
+        storyById: (id) => {
+          const storyContext = context.storyById(id);
+          return {
+            ...storyContext,
+            parameters: {
+              ...storyContext?.parameters,
+              docs: {
+                ...storyContext?.parameters?.docs,
+                theme: dark ? themes.dark : themes.light,
+              },
+            },
+          };
+        },
+      }}
+    >
+      {children}
+    </BaseContainer>
+  );
+};
```

Comment:
Ref: https://github.com/hipstersmoothie/storybook-dark-mode/issues/127

Reply:
lol what a reference

Reply:
This component is a solution for dark mode with docs, apparently, they are both addons from the storybook and docs are not currently working with dark mode...